dap_prinz_green_jobs/analysis/ojo_analysis/aggregate_by_soc.py

```diff
@@ -0,0 +1,82 @@
+from dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures import *
+from dap_prinz_green_jobs.getters.data_getters import save_to_s3
+from dap_prinz_green_jobs import BUCKET_NAME
+
+from datetime import datetime
+
+if __name__ == "__main__":
+    # date stamps as defined in https://github.com/nestauk/dap_prinz_green_jobs/issues/75
+
+    production = "True"
+    config = "base"
+    skills_date_stamp = "20230914"
+    occ_date_stamp = "20231002"
+    ind_date_stamp = "20231013"
```

Comment:
thinking about this - should we add this to an analysis config so we just need to update the dates in one place?